src/Sylius/Bundle/UiBundle/Resources/translations/messages.en.yml

```diff
@@ -58,6 +58,8 @@ sylius:
         manage_payment_method: Manage payment methods
         manage_product_option: Manage product option
         manage_shipping_destinations: Manage shipping destinations
+        manage_taxons: Manage taxons of your products
+        manage_taxation_of_your_products: Manage taxation of your products
```

Comment:
That is is already added [here](https://github.com/Sylius/Sylius/pull/4866/files#diff-700ac7d934d5990dad53037acb46869cR64)